src/shared/tpm2-util.c

```diff
@@ -1375,6 +1378,18 @@ int tpm2_primary_alg_from_string(const char *alg) {
         return -EINVAL;
 }
 
+TPM2Flags tpm2_flag_from_string(const char *flag) {
+        if (streq_ptr(flag, "pin"))
+                return TPM2_FLAGS_USE_PIN;
+        return -EINVAL;
```

Comment:
hmmm, so what string would you pass here to indicate "flag not set"?

Reply:
Hm, why would you need to do that? This is a bit of a left over from the approach with "feature keywords" anyway. The function is not used at the moment and probably not needed. I'll remove it.

Reply:
yeah, i didn't see it wasn't used anymore. please remove